src/app/controllers/Sell.class.ts

```diff
@@ -0,0 +1,39 @@
+import { SellInterface } from './../models/sell.model';
```

Comment:
I recommend renaming this file to `sell.class.ts` using lower case. In TypeScript, we follow the JavaScript style guides: https://google.github.io/styleguide/jsguide.html#file-name